### ISSUE TITLE
VERSION output of weaveworks/ignite-kernel:4.19.47 is incorrect

### ISSUE THREAD

**Issue by arun-gupta** [weaveworks/ignite#158]
Created and started a VM as:

```
ignite create weaveworks/ignite-ubuntu \
		--name my-vm \
		--cpus 2 \
		--memory 1GB \
		--size 6GB \
		--ssh
ignite vm start my-vm
```

`VERSION` output of `ignite kernels` is not shown correctly. See the output below:

```
ignite kernels
KERNEL ID		NAME					CREATED		SIZE	VERSION
f7fa9ca8e02ee44f	weaveworks/ignite-kernel:4.19.47	6m6s ago	49.0 MB	/bin/bash: strings: command not found
```

**Comment by luxas** [comment]
Thanks for the issue report!
It seems like the `strings` binary is not available. (test `which strings`)
We should make the kernel version detection more robust: #167

Does `apt-get install binutils` solve the issue for you?
If so, let's add that to the installation doc.

**Comment by luxas** [comment]
This is solved in the Go code #189 and the docs now https://github.com/weaveworks/ignite/blob/master/docs/installation.md#installing-dependencies

Cheers!
